### DISCUSSION TITLE
Większa liczba produktów w pakietach

### DISCUSSION THREAD

**Discussion by pgasidlo** [allegro/allegro-api#5882] [Suggestions - resolved]
:bulb: **Sugestia / Suggestions**

Testowałem wystawianie ofert przez nowe https://developer.allegro.pl/documentation#operation/createProductOffers. Działa. Mam jednak problem - narzucacie limit 5 produktów (GTINów) na ofertę. Czy można to zwiększyć? Mam aukcje, na których mam 6 (https://allegro.pl/oferta/pakiet-tom-1-6-chylka-remigiusz-mroz-10556199959), 7 (https://allegro.pl/oferta/pakiet-remigiusz-mroz-chylka-x7-testament-kasacja-11880286567) a nawet 15 produktów (https://allegro.pl/oferta/pakiet-joanna-chylka-skazanie-tom-1-15-r-mroz-11944705920).

**Comment by MaciejFrackowiak** [comment]
Dziękuję za Twoją sugestię. Przenoszę ją do Discussions, czyli miejsca odpowiedniego dla sugestii.

**Comment by MartaNowaczyk** [comment]
Ponownie weryfikujemy sugestie, dlatego wracam z informacją, że aktualnie w zestawie może być maksymalnie 10 różnych produktów. Temat oznaczam jako rozwiązany.
